src/data/java/dev/socketmods/socketnukes/datagen/utils/loottable/BaseLootTableProvider.java

```diff
@@ -3,6 +3,7 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import dev.socketmods.socketnukes.SocketNukes;
+import net.minecraft.data.CachedOutput;
 import net.minecraft.data.DataGenerator;
 import net.minecraft.data.DataProvider;
 import net.minecraft.data.HashCache;
```

Comment:
No need to import `HashCache`, its no longer used, neither is the `GSON` field in the class